Blazor_Foundation_6_Test_Shared/Documentation/Containers/ResponsiveAccordionTabs.razor

```diff
@@ -1,7 +1,105 @@
+@using Microsoft.AspNetCore.Components.Forms;
 
-@code {
 
+
+<SectionWrapper>
+    <SectionLeft>
+        <h2>Responsive Accordion Tabs</h2>
+        <p>
+            Added in 6.3.0, use the Markup of the Accordion or Tabs components to create Responsive Accordion Tabs.
+        </p>
+        <h2>Basics</h2>
+        <p>
+            Either the <NavLink href="/accordion">Accordion Markup</NavLink> or the <NavLink href="/tabs">Tabs Markup</NavLink>can be used to responsively switch between the two components at different breakpoints.
+            The Accordion should have an id specified, but the plugin will automatically generate one if the id is omitted.
+            Accordion content should also have an ID, or the # of the href should be specified, otherwise a random id will be generated
+        </p>
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="secondary">
+            The accordion/tabs values can be in any order.
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+
+        <h2>Accordion HTML Markup</h2>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
+                     Content="PEFjY29yZGlvbj4KICAgIDxBY2NvcmRpb25JdGVtIFRpdGxlPSJBY2NvcmRpb24gMSI+CiAgICAgICAgSSB3b3VsZCBzdGFydCBpbiB0aGUgb3BlbiBzdGF0ZSwgZHVlIHRvIHVzaW5nIHRoZSBgaXMtYWN0aXZlYCBzdGF0ZSBjbGFzcy4KICAgIDwvQWNjb3JkaW9uSXRlbT4KICAgIDwhLS0gLi4uIC0tPgo8L0FjY29yZGlvbj4=" />
+
+        <OpenCodeDev.Blazor.Foundation.Components.Containers.Callout TypeClass="secondary">
+            Once you put it all together, here's what you get!
+            Scale your browser down to see the toggle happen.
+            I am an <Code>Accordion on small and large</Code> but I am <Code>Tabs on medium</Code>
+        </OpenCodeDev.Blazor.Foundation.Components.Containers.Callout>
+        <Accordion>
+            <AccordionItem Title="Accordion 1">
+                <p>Panel 1. Lorem ipsum dolor</p>
+                <NavLink href="#">Nowhere to Go</NavLink>
+            </AccordionItem>
+            <AccordionItem Title="Accordion 2">
+                <p>Enter your card number</p>
+                <InputCreditCard @bind-Card="@Number" />
+                <Button Class="primary">I do nothing!</Button>
+            </AccordionItem>
+            <AccordionItem Title="Accordion 3">
+                <p>Some button</p>
+                <Button Class="primary">Primary</Button>
+                <Button Class="secondary">Secondary</Button>
+                <Button Class="success">Success</Button>
+                <Button Class="alert">Alert</Button>
+                <Button Class="warning">Warning</Button>
+            </AccordionItem>
+            <!-- ... -->
+        </Accordion>
+
+        <h2>Tabs HTML Markup</h2>
+        <HighlightCS Language="html" Full="true" HeaderIcon="mdi-language-html5" HeaderBGcolor="#5d2b90" HeaderTitle="Razor"
```

Comment:
i see you are using the external way here it is good maybe you should talk about The Accordion should have an id specified, but the plugin will automatically generate one if the id is omitted. Accordion content should also have an ID, or the # of the href should be specified, otherwise a random id will be generated

Note: Random Id is not generated automatically so remove that statement, user must use an id.